ui/src/common/constants.js

```diff
@@ -21,8 +21,22 @@ export const LITERATURE_PID_TYPE = 'literature';
 export const JOBS_PID_TYPE = 'jobs';
 export const CONFERENCES_PID_TYPE = 'conferences';
 
+export const INVENIO_URL = '//invenio-software.org';
+export const HOLDINGPEN_URL = '/holdingpen';
+export const AUTHORLIST_TOOL_URL = '/toos/authorlist';
+export const INSPIRE_TWITTER_ACCOUNT = 'https://twitter.com/inspirehep';
 export const SURVEY_LINK = 'https://forms.gle/CMgi3UsbpMuXxAfr6';
 export const FEEDBACK_EMAIL = 'feedback@inspirehep.net';
+export const BLOG_URL = 'https://blog.inspirehep.net';
+export const HELP_BLOG_URL = 'https://labs.inspirehep.net/help';
```

Comment:
do we want this hardcoded? we'll change soon to ``inspirehep.net``

Reply:
well this is one of the step to update it, I moved it to a single place.